uber/templates/preregistration/form.html

```diff
@@ -107,7 +107,7 @@
     <div class="form-group">
         <label for="badges" class="col-sm-2 control-label">Badges</label>
         <div class="col-sm-6">
```

Comment:
This function doesn't exist anymore.  It was from the anthrocon codebase, and unfortunately we couldn't use it directly because we it had a bunch of stuff hard-coded.  Our table prices vary depending on which table, so if we want to present a live-updating list of the current price of all the things they've selected, we'll need to either re-implement the same basic logic client-side or make an ajax request to the server to get it.  In the meantime I'm just removing this to prevent a JS error.